app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -358,207 +361,231 @@ class YearToDateLiabilityController @Inject() (
     }
 
   def estimatedIncome(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
-    withFillingOutReturnAndYTDLiabilityAnswers(request) { (_, fillingOutReturn, answers) =>
-      (answers, fillingOutReturn.draftReturn) match {
-        case (calculatedAnswers: CalculatedYTDAnswers, draftReturn: DraftSingleDisposalReturn) =>
-          withDisposalDate(draftReturn) { disposalDate =>
-            commonDisplayBehaviour(calculatedAnswers)(
-              form = _.fold(
-                _.estimatedIncome.fold(estimatedIncomeForm)(a => estimatedIncomeForm.fill(a.inPounds())),
-                c => estimatedIncomeForm.fill(c.estimatedIncome.inPounds())
+    withFillingOutReturnAndYTDLiabilityAnswers(request) {
+      case (_, fillingOutReturn, _) if isATrust(fillingOutReturn) =>
```

Comment:
you don't need this `case` here or in similar places in this controller - you take take them back out the way it was before

Reply:
Ok, done.